Specifications/Language/3_Expressions/ValueLiterals.md

```diff
@@ -20,11 +20,13 @@ Value literals for the [`Int` type](https://github.com/microsoft/qsharp-language
 
 ## BigInt Literals
 
-Value literals for the [`BigInt` type](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types) are always postfixed with `L` and can be expressed in decimal or hexadecimal representation. Literals expressed in hexadecimal are prefixed with `0x`. There is no prefix for the commonly used decimal representation.
+Value literals for the [`BigInt` type](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types) are always postfixed with `L` and can be expressed in decimal, binary, octal or hexadecimal representation. Literals expressed in binary are prefixed with '0b', in octal prefixed with '0o', in hexadecimal prefixed with `0x`. There is no prefix for the commonly used decimal representation.
```

Comment:
> Literals expressed in binary are prefixed with '0b', in octal prefixed with '0o', in hexadecimal prefixed with `0x`.

Also related to consistency, the Int literal section above uses the format "with `0o` for octal, and with `0x` for hexadecimal", can we keep that format here as well? By the way, the backticks around `` `0x` `` format it in a monospace font, so I think it'd be good to keep that also, instead of using single quotes around '0b' and '0x'.

Reply:
addressed